pkg/vm/json_test.go

```diff
@@ -303,12 +303,12 @@ func execStep(t *testing.T, v *VM, step vmUTStep) {
 }
 
 func (v *vmUTState) UnmarshalJSON(data []byte) error {
-	switch s := string(data); s {
-	case `"Break"`:
+	switch s := string(data); strings.ToUpper(s) {
+	case `"BREAK"`:
 		*v = vmUTState(breakState)
-	case `"Fault"`:
+	case `"FAULT"`:
 		*v = vmUTState(faultState)
-	case `"Halt"`:
+	case `"HALT"`:
 		*v = vmUTState(haltState)
```

Comment:
We have proper `(*State).UnmarshalJSON`, why not use it here?

Reply:
Fixed.